dev-docs/source/Logging.rst

```diff
@@ -126,6 +126,26 @@ In general:
   from mantid.kernel import logger
   logger.warning('this is a custom warning')
 
+Using :code:`logging` in Python
```

Comment:
Can you add a link to/form `docs/source/api/python/mantid/kernel/Logger.rst` in this as well. That documentation is auto-generated and not nearly as useful.

Reply:
The link here should be
```rst
:class:`mantid.kernel.Logger`
```

Reply:
Ok. I thought you meant to link the other way round.